Assets/Scripts/Controllers/UIController.cs

```diff
@@ -1,23 +0,0 @@
-﻿#region License
```

Comment:
Question: this class was apparently added for future extensibility, but so far does nothing. Should it be removed now? Here are the commits which touch it:
```
$ git slog -- Assets/Scripts/Controllers/UIController.cs
* [2016-09-16] 6940a61  Organizes the controller folder (Ben Pearson)
* [2016-08-27] 8257d1e  UIController.cs SCS (#893) (vringar)
* [2016-08-22] 73b8d8b  adds headers (#600) (Zak)
* [2016-08-20] 7457090  UI Controller (#420) (Jack Evans)
```

The full log on the first one is
```
commit 7457090957d4923cf743751e898f5a37adf1eb6b
Author: Jack Evans <j-j-evans@hotmail.com>
Date:   Sat Aug 20 18:25:27 2016 +0100

    UI Controller (#420)

    * Created UI Controller for future code driven UI.

    * Instantiated UI Controller

    UI controller now instantiated by WorldController to fit with
    https://github.com/TeamPorcupine/ProjectPorcupine/issues/403
```